src/assets/scss/helpers/_animations.scss

```
@@ -127,3 +127,24 @@
     }
   }
 }
+
+/* pulse validate checkbox */
+/* Aplica en .error [role="checkbox"] > input {*/
+/*animation: pulse-checkbox 0.4s ease-in-out infinite;*/
+
+@keyframes pulse-checkbox {
```

```suggestion
@keyframes pulse {
```
@GusBaamonde Que te parece llamarlo así? Y la animación que voy a agregar [aquí ](https://github.com/FromDoppler/doppler-editors-webapp/pull/431/files#r1116987142) le llamo `pulse-secondary` está burdo el nombre pero marca un patrón para cuando se agreguen animaciones parecidas a _pulse_ y para cuando diseñen porque creo que esto será útil marcarle algo del camino para los diseñadores, , me los imagino revisando en la Style las animaciones disponibles.

Se te ocurre un mejor nombre @andresmoschini

---

no tengo opinión, estoy bastante fuera de tema.

---

@Cromeror creo que igual el tema de la animación, en este componente por el momento quedo fuera de esta tarea. Podemos verlo en otra tarea o en https://github.com/FromDoppler/doppler-editors-webapp/pull/431